### ISSUE TITLE
Make it possible to select spawn point

### ISSUE THREAD

**Issue by Gireen** [Unvanquished/Unvanquished#1232]
It would be nice to select a preferred telenode/egg on the map to spawn from.

**Comment by DolceTriade** [comment]
Yeah we've always wanted this :( It probably requires some CONFIG string stuff and a UI

**Comment by superdupont-fr** [comment]
How do you think the player choose the Telenod/Egg ?
List all available spots?
Indicate the room (if you know the map), or if there is or not many death or enemies?
When there is only one? When there is 20 spots?

**Comment by Gireen** [comment]
With a big "minimap" #1216 where spawns are selectable .
One could maybe also just define a point and automatically use the closest spawn.

**Comment by bmorel** [comment]
I'm not sure a minimap would do the job on many maps, which are 3D mazes.

One (relatively recent) game I can think about which had such kind of problematic is «mordheim, city of the damned». To place units there, you can cycle through the spawn points and validate one. This might be more appropriated, despite the choice being a bit slower to do. It might also be easier to implement though, as all pieces are already done.

**Comment by bmorel** [comment]
One point that I believe was not said out loud yet: the game currently merges close enough buildings in a single spot.
Instead of being able to select exactly which spawn point one wants to pop at, it would be much easier to build an interface to select which *outpost* to spawn at, when there is a spawn, of course.
One way to make this fast for advanced players (for teleports idea in [gameplay#5](https://github.com/Unvanquished/gameplay/issues/5), for example) is to use the same method text-web-browsers do: give valid destination outposts a number.

**Comment by DolceTriade** [comment]
To start out with, a menu and the map location would be a good start. Perhaps a list of spawns and their distances from your death might be another option. Other than that, a minimap with some beacons and stuff on it would be the ideal.
